internal/hook/item_hook_handler.go

```diff
@@ -233,14 +233,14 @@ func (h *DefaultItemHookHandler) HandleHooks(
 			},
 		)
 
-		hookTracker.Record(namespace, name, hookFromAnnotations.Container, HookSourceAnnotation, "", phase, false)
-		if err := h.PodCommandExecutor.ExecutePodCommand(hookLog, obj.UnstructuredContent(), namespace, name, "<from-annotation>", hookFromAnnotations); err != nil {
+		hookFailed := false
+		if err = h.PodCommandExecutor.ExecutePodCommand(hookLog, obj.UnstructuredContent(), namespace, name, "<from-annotation>", hookFromAnnotations); err != nil {
```

Comment:
trivial: it'd be better to limit the scope of the var `err` to the `if` clause and use `hookFailed` in line 242

Reply:
Sure, let me make it more clear.